src/Analyzers/VisualBasic/Analyzers/IncludeParenthesesForMethodInvocations/VisualBasicIncludeParenthesesForMethodInvocationsDiagnosticAnalyzer.vb

```diff
@@ -0,0 +1,58 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis.Diagnostics
+Imports Microsoft.CodeAnalysis.VisualBasic.Precedence
+Imports Microsoft.CodeAnalysis.VisualBasic.LanguageServices
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Microsoft.CodeAnalysis.RemoveUnnecessaryParentheses
+Imports Microsoft.CodeAnalysis.Precedence
+Imports Microsoft.CodeAnalysis.LanguageServices
+Imports Microsoft.CodeAnalysis.CodeStyle
+Imports System.Collections.Immutable
+Imports Microsoft.CodeAnalysis.Options
+Imports Microsoft.CodeAnalysis.VisualBasic.CodeStyle
+Imports Microsoft.CodeAnalysis.Operations
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.ParenthesesStyle
+    <DiagnosticAnalyzer(LanguageNames.VisualBasic)>
+    Friend NotInheritable Class VisualBasicIncludeParenthesesForMethodInvocationsDiagnosticAnalyzer
+        Inherits AbstractBuiltInCodeStyleDiagnosticAnalyzer
+
+        Private Shared ReadOnly s_descriptorForInvocations As DiagnosticDescriptor = CreateDescriptorWithId(IDEDiagnosticIds.IncludeParenthesesForMethodInvocationsDiagnosticId,
+            New LocalizableResourceString(NameOf(VisualBasicAnalyzersResources.Add_invocation_parentheses), VisualBasicAnalyzersResources.ResourceManager, GetType(VisualBasicAnalyzersResources)),
+            New LocalizableResourceString(NameOf(VisualBasicAnalyzersResources.Add_invocation_parentheses), VisualBasicAnalyzersResources.ResourceManager, GetType(VisualBasicAnalyzersResources)),
+            isUnnecessary:=True)
```

Comment:
@CyrusNajmabadi, From my understanding:

- If `visual_basic_style_parentheses_for_invocations` is true, we suggest to include paren when they don't exist.
- If `visual_basic_style_parentheses_for_invocations` is false, we suggest to delete paren when they exist.

I'm putting `isUnnecessary=True` to fade the paren when they exist and the option is set to `False`, but what would happen if the option is set to True?